src/insights_client/__init__.py

```diff
@@ -301,47 +300,46 @@ def update_motd_message():
         if not os.path.lexists(MOTD_FILE):
             logger.debug(
                 ".registered and .unregistered do not exist; "
-                "pointing the MOTD file '{source}' to '{motd}'".format(
-                    source=MOTD_SRC, motd=MOTD_FILE
-                )
+                "pointing the MOTD file '%s' to '%s'",
+                MOTD_SRC,
+                MOTD_FILE
```

Comment:
What does this change achieve?

Reply:
It unifies the log formatting syntax, questioned by your later [comment](https://github.com/RedHatInsights/insights-client/pull/194#discussion_r1549333579). It also leverages the native logging module interpolation.